### PR TITLE
support golang integration test

### PR DESCRIPTION
**Pull request by kfangw**
### Github Issue
#471 
What you need to do to add integration testcase is just
adding `xxx_test.go` in `./tests/client`

### CODE REVIEW

**Comment by kfangw** [comment]
You can write an integration test case by using GoLang.
Please review.

File: tests/client/entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -xe
+
+if [ $# -eq 1 ]; then
```
Comment (Geod24) [inline tests/client/entrypoint.sh]: Does the go syntax support running multiple tests ? E.g. `./sdb_test -test.run test1 test2` ?

**Comment by anarcher** [comment]
Wow! It’s amazing! :-)

File: tests/client/entrypoint.sh (hunk 1 above)
Comment (kfangw) [reply tests/client/entrypoint.sh]: I did not test yet. But it supports regex.

Comment (kfangw) [reply tests/client/entrypoint.sh]: When I tested, It does not support multiple arguments.
But If you want to test multiple, then, using regexp will helpful
```
  -test.run regexp
    	run only tests and examples matching regexp
```

**Comment by kfangw** [comment]
This PR is pretty obvious so I merge it without approvement.

**Pull request merged**